core/modules/wbase/Base.cc

```diff
@@ -373,4 +319,4 @@ void StringBuffer2::_setSize(unsigned size) {
     _bufferSize = size;
 }
 
-}}} // namespace lsst::qserv::wbase
+}}} // namespace
```

Comment:
let's keep it how it was please :). We don't create / rename these names often, there is value in keeping the names